server/etcdserver/api/etcdhttp/health.go

```diff
@@ -111,18 +113,57 @@ var (
 		Name:      "health_failures",
 		Help:      "The total number of failed health checks",
 	})
```

Comment:
I feel people will be confused between `etcd_server_health_success` and `etcd_server_root_health`. 

How about putting deprecation notice on the former one?

Reply:
done